rovercode/app.py

```diff
@@ -0,0 +1,181 @@
+"""Rovercode app."""
+import websocket
+from threading import Thread
+import logging
+import time
+import json
+from oauthlib.oauth2 import BackendApplicationClient
+from requests_oauthlib import OAuth2Session
+from dotenv import load_dotenv
+import os
+
+logging.basicConfig()
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.getLevelName('INFO'))
+
+import constants
+from binary_sensor import BinarySensor
+from motor_controller import MotorController
+from drivers.dummy_binary_sensor import DummyBinarySensor
+from drivers.adafruit_pwm_manager import AdafruitPwmManager
+from drivers.VCNL4010 import VCNL4010
+
+
+def init_inputs(rover_params, dummy=False):
+    """Initialize input GPIO."""
+    binary_sensors = []
+
+    def get_env_int(name):
+        try:
+            return int(os.getenv(name, None))
+        except Exception:
+            return None
```

Comment:
It might be helpful to print an error message here since `name` isn't going to be configured like the user might expect

Reply:
Good idea.

Reply:
I don't think that this would be for an unknown variable. It would be if the value is not an `int`

Reply:
You might want to add the value that didn't work as well

Reply:
Also, it would happen if `name` isn't in the environment

Reply:
Done.